test/AutoDiff/Sema/transpose_attr_type_checking.swift

```diff
@@ -550,6 +554,7 @@ extension Struct where T: Differentiable & AdditiveArithmetic {
 // Check that `@transpose` attribute rejects stored property original declarations.
 
 struct StoredProperty: Differentiable & AdditiveArithmetic {
+  // expected-note @+1 {{candidate getter does not have expected type '(StoredProperty) -> () -> StoredProperty'}}
   var stored: Float
```

Comment:
Note: it's nice that this note mentions `candidate *getter* does not have expected type ...` instead of just `candidate does not have expected type ...`, so that it clearly refers to the stored property getter's type, not the stored property's type.